lib/oelite/profiling.py

```diff
@@ -98,9 +97,10 @@ def print_deferred(self):
         for p in self.deferred:
             p.print_delta()
 
-    def __init__(self, name):
+    def __init__(self, name, print_walltime=True):
```

Comment:
you are changing profiling.py in a commit that only mentions baker.py. Shouldn't it be split up?